src/data/markdown/docs/01 guides/02 Using k6/08 Tags and Groups.md

```diff
@@ -24,13 +24,14 @@ allowing you the BDD-style of testing.
 import { group } from 'k6';
 
 export default function () {
-  group('user flow: returning user', function () {
-    group('visit homepage', function () {
-      // load homepage resources
-    });
-    group('login', function () {
-      // perform login
-    });
+  group('add several products to the shopping cart', function () {
+    // ...
+  });
+  group('go to login and authenticate', function () {
+    // ...
+  });
+  group('checkout process', function () {
+    // ...
```

Comment:
I am for at least one example with nested groups, but it should probably be not an anti-pattern one ;)

Reply:
I intentionally removed it to simplify the example. 

Not saying that is the best option.

It is mentioned in the description.